project/MimaExcludes.scala

```diff
@@ -36,8 +36,8 @@ object MimaExcludes {
 
   // Exclude rules for 3.0.x
   lazy val v30excludes = v24excludes ++ Seq(
-    // [SPARK-28091[CORE] Extend Spark metrics system with user-defined metrics using executor plugins
-    ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.ExecutorPlugin.init"),
+    // [SPARK-29399][core] Remove old ExecutorPlugin interface.
+    ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.ExecutorPlugin"),
```

Comment:
Why this removal is not reported in MIMA? This API was added in Spark 2.4

Reply:
cc @HyukjinKwon Do you have an idea?

Reply:
If you mean `org.apache.spark.ExecutorPluginContext`, that's not in Spark 2.4. Seems `org.apache.spark.ExecutorPlugin` is properly excluded here.

Reply:
I see. Thanks!